documentation/asciidoc/computers/getting-started/setting-up.adoc

```diff
@@ -68,7 +68,7 @@ image::images/peripherals/sd-card.png[alt="Inserting a microSD card into a Raspb
 
 [[recommended-capacity]]
 
-We recommend using an SD card with at least 16GB of storage with Raspberry Pi OS. If you use Raspberry Pi OS Lite, we recommend at least 4GB.
+We recommend using an SD card with at least 16GB of storage for Raspberry Pi OS installations. For Raspberry Pi OS Lite, we recommend at least 4GB. You can use any SD card whose capacity exceeds this minimum size; there is no maximum size. As with any other boot media, you'll see improved performance on SD cards with faster read and write speeds.
```

Comment:
Probably want to bump that to 32GB and 16GB, that was written a long time ago. Also we might want to be a bit less direct that "there is no maximum size" because when 2D cards hit 2TB (if they can) then there will be, but it'll be a software limit rather than a hardware limit.

Reply:
[Should we be worried?](https://www.tomsguide.com/news/the-worlds-first-2tb-microsd-card-is-here-what-you-need-to-know)

Reply:
```suggestion
We recommend using an SD card with at least 32GB of storage for Raspberry Pi OS installations. For Raspberry Pi OS Lite, we recommend at least 16GB. You can use any SD card with a capacity of less than 1.5TB. As with any other boot media, you'll see improved performance on SD cards with faster read and write speeds.
```
Added your minimum size tweaks and a simple version of @tdewey-rpi's comment below. I'm not sure if the Getting Started page is the right place to discuss partition tables, but if others feel the information should live here, I'm open to it.